### DISCUSSION TITLE
"Darstellung" Ventilöffnungsgrad: HM-CC-RT-DN (VALVE_STATE) vs. eTRV-x (LEVEL)

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#633] [Q&A] [answered]
Mir ist heute mal aufgefallen das sich die Darstellung der obigen Geräte bezüglich "Ventilöffnungsgrad" unterscheidet.
**HM-CC-RT-DN (`VALVE_STATE`):**
![HA_HM-CC-RT-DN_VALVE_STATE_Gerät_v2](https://user-images.githubusercontent.com/54898249/193100675-1bea18d9-e064-4e7d-ae0a-bd0383e14119.JPG)
**eTRV-x (`LEVEL`):**
![HA_eTRV_LEVEL_Gerät_v2](https://user-images.githubusercontent.com/54898249/193100796-93b440cc-1b07-429b-8809-2f84403b8d51.JPG)

Das eTRV fällt hier aus dem Rahmen weil der Ventilöffnungsgrad unter "Konfiguration" gelistet ist und als Eingabe/Auswahlfeld dargestellt wird.

Woher das kommt ist klar. 
eQ-3 ist der Meinung der `LEVEL` eines eTRV ist beschreibbar. Kann man sogar über WebUI-Programme machen. 
"Beschreibbar" ja, funktioniert auch fehlerfrei nur das eTRV bleibt völlig ungerührt. Es "bewegt" sich kein bisschen. Also nutzlos.

Ich würde es gut finden den `LEVEL` von eTRV's wie bei `VALVE_STATE` von HM-Thermostaten anzulegen.
Also als Sensor ohne Interaktionsmöglichkeit.

Spricht was dagegen (außer Arbeit für den Maintainer ;-) )

**Comment by SukramJ** [comment]
> fehlerfrei nur das eTRV bleibt völlig ungerührt

Ich hoffe Du hast noch Garantie. Mein eTRV-2 funktoniert problemlos.

Daher werde ich nichts ändern.

**Reply by Baxxy13** [reply]
Punkt für Dich. 
Das Teil hatte eine ungeahnte Batterieschwäche.
Frisch betankt fährt es nun tatsächlich auf den Wert den ich einstelle um dann aber innerhalb kürzester Zeit wieder "gegenzusteuern".
Die interne Regelungslogik lässt sich damit also nicht überlisten.
Ich ändere mein Statement von "Also nutzlos" in relativ nutzlos.

**Comment by Baxxy13** [comment, marked as answer]
Ab 1.21.1 wird der Ventilöffnungsgrad (`LEVEL`) von IP-Heizkörperthermostaten standardmäßig als Sensor integriert.
Damit ist die Integration und Darstellung einheitlich mit den HM-Heizkörperthermostaten und das Thema für mich beantwortet.
